Sources/SwiftUIPager/Pager+Buildable.swift

```diff
@@ -30,6 +30,11 @@ extension Pager: Buildable {
             .mutating(keyPath: \.contentLoadingPolicy, value: .eager)
     }
 
+    /// User can only swipe forward so in one direction
+    public func forwardOnly(_ enabled: Bool = false) -> Self {
```

Comment:
can you move this inside the `#if !os(tvOS)`? this won't have any purpose in that platform as there's no `DragGesture`

Reply:
done